plugins/@grouparoo/iterable/src/lib/export/destinationMappingOptions.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  DestinationMappingOptionsMethod,
+  DestinationMappingOptionsResponseTypes,
+} from "@grouparoo/core";
+import { connect } from "./../connect";
+
+export const destinationMappingOptions: DestinationMappingOptionsMethod = async ({
+  appOptions,
+}) => {
+  const client = await connect(appOptions);
+  const required = getRequiredFields();
+  const known = await getUserFields(client);
+
+  return {
+    labels: {
+      property: {
+        singular: "Iterable Contact Property",
+        plural: "Iterable Contact Properties",
+      },
+      group: {
+        singular: "Iterable List",
+        plural: "Iterable Lists",
+      },
+    },
+    properties: {
+      required: required,
+      known: known,
+      allowOptionalFromProperties: true,
+    },
+  };
+};
+
+export const getRequiredFields = (): Array<{
+  key: string;
+  type: DestinationMappingOptionsResponseTypes;
+}> => {
+  return [{ key: "email", type: "string" }];
+};
+
+const mapTypesFromIterableToGrouparoo = (iterableType) => {
+  const map = {
+    string: "string",
+    long: "integer",
+    date: "date",
+    boolean: "boolean",
+  };
+  const grouparooType = map[iterableType];
+  if (!grouparooType) {
+    throw `Unknown iterable type: ${iterableType}`;
```

Comment:
we found it's better to `log` and allow this function to return `undefined` or `null`